### DISCUSSION TITLE
Can I configure the initial dokku setup without accessing the dokku setup page on my ip?

### DISCUSSION THREAD

**Discussion by Axedyson** [dokku/dokku#5266] [Q&A] [answered]
I would like to configure the initial dokku setup without having to use a gui interface, i.e. I would like to configure dokku programmatically. Is that possible, simply by issuing some dokku commands? The reason why I would like to do this is because I use and prefer terraform (more specifically [remote-exec](https://www.terraform.io/language/resources/provisioners/remote-exec)) to configure dokku

**Comment by josegonzalez** [comment, marked as answer]
Newer versions of dokku do not start a web ui for the setup (that was removed in 0.25.0). You can manually install Dokku via [these instructions](https://dokku.com/docs/getting-started/install/debian/) (which do essentially the same thing as the bootstrap script).
